s3_tools/read.py

```diff
@@ -36,12 +33,14 @@ def read_object_to_bytes(bucket: str, key: str) -> Optional[bytes]:
     session = boto3.session.Session()
     s3 = session.client("s3")
     obj = s3.get_object(Bucket=bucket, Key=key)
-    data = obj["Body"].read() if "Body" in obj else None
 
-    return data
+    if "Body" not in obj:
+        raise KeyError("Missing key 'Body' on response")
```

Comment:
We can enrich our docstring with a new `Raises` section. [Numpy Style Guide](https://numpydoc.readthedocs.io/en/latest/format.html) 

The next step is to create our custom `Exceptions` right? :slightly_smiling_face:

Reply:
Added the exceptions to docstring. We can discuss the exceptions and how to deal with them. I have no ideas for now.